### PR TITLE
NAS-129944 / 24.10 / Add new badge indicators

### PR DESCRIPTION
**Pull request by denysbutenko**
- Added new page indicators for Dashboard and Enclosure pages.
- Added new feature indicator to Configure button on dashboard.
- Also, removed `FlexModule` dependency from `PageHeaderModule`

### CODE REVIEW

File: src/app/pages/dashboard/components/dashboard/dashboard.component.ts [hunk 1]
```diff
@@ -42,6 +42,10 @@ export class DashboardComponent implements OnInit {
   readonly savedGroups = toSignal(this.dashboardStore.groups$);
   readonly isLoading = toSignal(this.dashboardStore.isLoading$);
   readonly isLoadingFirstTime = computed(() => this.isLoading() && this.savedGroups() === null);
+  readonly newFeatureConfig = {
+    key: 'dashboardConfigure',
+    message: this.translate.instant('Try out the new dashboard configuration feature!'),
```
Comment (undsoft) [inline src/app/pages/dashboard/components/dashboard/dashboard.component.ts:47]: Let's remove the exclamation mark.

**Review by undsoft — APPROVED** [review]
Okay, I think new circle can be improved, but I'll create a separate ticket for it.
This one looks good.

**Review by RehanY147 — CHANGES_REQUESTED** [review]
Should we rename the `"New Enclosure"` to just `"Enclosure"` and `"Enclosure"` to `"Enclosure (old)"`.

**Comment by denysbutenko** [comment]
> Should we rename the `"New Enclosure"` to just `"Enclosure"` and `"Enclosure"` to `"Enclosure (old)"`.

This wasn't mentioned yet but I'm okay to do if necessary.

**Review by RehanY147 — APPROVED** [review]

**Pull request merged**